### PR TITLE
[KYUUBI #2072] Improve rest server behavior

### PR DESCRIPTION
**Pull request by ulysses-you**
### _Why are the changes needed?_

1. use host address rather than host name, since we also use host address in thrift server
2. remove alert in web ui, it's noisy..

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by ulysses-you** [comment]
cc @cfmcgrady @yaooqinn

**Comment by yaooqinn** [comment]
please fix the PR title to be more specific and triage PR by labels

**Review by yaooqinn — APPROVED** [review]

**Review by cfmcgrady — APPROVED** [review]

File: kyuubi-server/src/main/resources/org/apache/kyuubi/ui/static/kyuubi.js [hunk 1]
```diff
@@ -43,5 +43,4 @@ $(function(){
 		});
 	}
 	$('.item.server.version.right.padded').val(kyuubiVersion)
-	alert(kyuubiVersion);
```
Comment (SteNicholas) [inline kyuubi-server/src/main/resources/org/apache/kyuubi/ui/static/kyuubi.js:46]: It's better to replace this with `console.log(kyuubiVersion)`.

Comment (ulysses-you) [reply kyuubi-server/src/main/resources/org/apache/kyuubi/ui/static/kyuubi.js:46]: it's a common function and will be invoked in many places..

**Comment by yaooqinn** [comment]
thanks, merged to master
